tests/connect/compression_error-002.phpt

```diff
@@ -0,0 +1,19 @@
+--TEST--
+MongoDB\Driver\Manager: Connecting with invalid compressor values
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php NEEDS('STANDALONE'); CLEANUP(STANDALONE); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+echo throws(function() {
+	$manager = new MongoDB\Driver\Manager(STANDALONE, [ 'compressors' => "foo\xFEbar"] );
+}, 'MongoDB\Driver\Exception\UnexpectedValueException'), "\n";
+?>
+===DONE===
+<?php exit(0); ?>
+--EXPECTF--
+OK: Got MongoDB\Driver\Exception\UnexpectedValueException
+Detected invalid UTF-8 for fieldname "compressors": foo�bar
```

Comment:
As-is, this can use `EXPECT` instead of `EXPECTF`. If you want to be consistent with other UTF-8 error tests, we tend to use `: %s` at the end instead of repeating the input.